### PR TITLE
 fetching  data from API from and styling 

### PR DESCRIPTION
**Pull request by wolterbwambale**
### In this project, i have implemented the following:-

- [ ] Home page
      When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by  the category name.      
- [ ] Along with the category name it displays some numeric values per category that come from the API.

- [ ]  When the user clicks (or taps) on a category item, the application navigates to the details page.
- [ ] Details page
- [ ] In the details page, the webapp retrieves data from the API to show detailed data in the category.
- [ ]  When the user clicks on the "Back" button (<), the user navigates to the home page

   

- [ ]  Create unit tests for pure functions in your Redux code.
- [ ] Create integration tests for your application using the React Testing Library

- [ ] The project is a single-page application (SPA) built with React and Redux.
- [ ] The data retrieved from the API should be stored in the Redux store.
- [ ] You should filter the data that you retrieve from the API using a Filter stateless component.
- [ ]  Add a link to the online version to README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+>[Fontawsome](https://fontawesome.com/) Special thanks to fontawsome for the icons assets.
+
+> Give credit to everyone who inspired your codebase.
+  Thanks to the Microverse team for the great curriculum.
+  Thanks to the Code Reviewer(s) for the insightful feedbacks.
+  A great thanks to My coding partner(s), morning session team, and standup team for their contributions.
+  Hat tip to anyone whose code was used.
+
```
Comment (emmanuellekamwa) [inline README.md:163]: - [ ] You have to give credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ⛔❌**
Hello @wolterbwambale 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Descriptive PR.
- Good commit messages.
- Correct git-flow.

## Required Changes ♻️
_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thank the Microverse community for their support.
+
+ I would like to thank Nelson Sakwa . the author of the original design
```
Comment (wayungi) [inline README.md:162]: - [ OPTIONAL] Good work giving credit where its due. I would suggest that you make **Nelson Sakwa** & **original design** links to te respective entities 😄

**Review by wayungi — APPROVED** [review]
Hi @wolterbwambale , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] Well written READ ME ✔️ 
- [ ] All Linter checks are passing ✔️ 
- [ ] Requested change has been made ✔️ 
- [ ] Project up and running ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**